rotkehlchen/globaldb/counterparty_mappings.py

```diff
@@ -0,0 +1,15 @@
+from rotkehlchen.errors.asset import UnknownAsset
+from rotkehlchen.globaldb.handler import GlobalDBHandler
+
+
+def get_asset_id_by_counterparty(symbol: str, counterparty: str) -> str:
+    with GlobalDBHandler().conn.read_ctx() as cursor:
+        cursor.execute(
+            'SELECT local_id FROM counterparty_asset_mappings WHERE '
+            'symbol=? AND counterparty=?',
+            (symbol, counterparty),
+        )
+        if (result := cursor.fetchone()) is not None:
+            return result[0]
+
+        raise UnknownAsset(f'Cannot find {symbol} for counterparty {counterparty}')
```

Comment:
This is not just unknown asset. 

Can it not also be unknown counterparty?
Can it not also be just no mapping existing for that combination?